src/about.html

```diff
@@ -10,27 +10,26 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <title>ESFiddle</title>
+    <script src="https://use.fontawesome.com/52284be701.js"></script>
```

Comment:
Actually, I did this in the head because it's what font awesome tells you to do, but it's asynchronous so it won't block the loading of the page